jetpack-start/wp-cli-api.php

```diff
@@ -235,8 +235,8 @@ private function maybe_create_user() {
 	}
 
 	private function run_jetpack_bin( $script, $args = array() ) {
-		// Running local versions for now until https://github.com/Automattic/jetpack/pull/7294 lands
-		$script_path = __DIR__ . '/bin/' . $script;
+		// Use Jetpack's bin scripts
+		$script_path = WPMU_PLUGIN_DIR . '/jetpack/bin/' . $script;
```

Comment:
Hmm, this could cause troubles though in the future if a site is pinned to an older version, and the latest JP version/default has slightly different connection logic. Edge case, but could be messy to connect through a newer version and then proceed to use an older version right after.

Reply:
If instead of using the default version we'd use the pinned version, we'd avoid that problem right?